config/src/caching.rs

```diff
@@ -22,7 +22,7 @@ impl StorageCachingConfig {
     /// Whether caching should be enabled for the chain id
     pub fn enable_for_chain_id(&self, chain_id: u64) -> bool {
         // ignore dev chain
-        if chain_id == 1337 {
+        if chain_id == 31337 {
```

Comment:
this should be extended to `&[31337, 1337, 99].contains(chain_id)`, because we want to ignore dev chains generally.